operator/cmd/mesh/testdata/manifest-generate/output/all_on.golden-show-in-gh-pull-request.yaml

```diff
@@ -5358,7 +5358,13 @@ data:
           failureThreshold: {{ annotation .ObjectMeta `readiness.status.sidecar.istio.io/failureThreshold` .Values.global.proxy.readinessFailureThreshold }}
         {{ end -}}
         securityContext:
-          allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
+          {{ if or (eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY`)  .Values.global.proxy.privileged -}}
+          allowPrivilegeEscalation: true
+          privileged: true
```

Comment:
Correct, CAP_NET_ADMIN is sufficient.

Reply:
I need to rewrite route local net, if no privilege enabled, `sysctl -w net.ipv4.conf.eth0.route_localnet=1` will fail. `Readonly file system`

Reply:
Aren't you executing that in the `istio-init` container though? This part is changing the `istio-proxy` container's `securityContext`

Reply:
Yeah, but still get error when only add CAP_NET_ADMIN. It can be tested via docker run https://docs.docker.com/engine/reference/run/#runtime-privilege-and-linux-capabilities

Reply:
What I'm trying to say is: the `sysctl` exection happens in the `istio-init` container, this change is affecting the `istio-proxy` container

Reply:
I must be wrong, not sure how the ci checks it.

Reply:
Then it may be due to the use of the [original src filter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/listener/original_src/v3/original_src.proto#envoy-v3-api-msg-extensions-filters-listener-original-src-v3-originalsrc). That may require `CAP_NET_RAW`. In any case, it's definitely not OK to run it as privileged just because we don't know yet what capabilities are needed to use that feature.

Reply:
@rlenglet Right, i am meant to change the istio-init container, forget to fix it yesterday, will update instantly.

Reply:
fixed